app/api/server/v1/email-inbox.js

```diff
@@ -19,21 +19,31 @@ API.v1.addRoute('email-inbox.list', { authRequired: true }, {
 	},
 });
 
+
 API.v1.addRoute('email-inbox', { authRequired: true }, {
 	get() {
 		try {
 			check(this.queryParams, {
 				_id: Match.Maybe(String),
+				email: Match.Maybe(String),
 			});
-			const { _id } = this.queryParams;
+			const { _id, email } = this.queryParams;
```

Comment:
It's not a good practice using the same `get` method to fetch a record by `_id` and to search for data with other parameters.

My suggestion is to create a new get endpoint `email-inbox.search` and pass the term(email) as a query parameter.